usr/src/uts/common/io/scsi/adapters/smartpqi/smartpqi.conf

```diff
@@ -0,0 +1,25 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+#
+# Copyright 2018 Nexenta Systems, Inc.  All rights reserved.
+#
+
+#
+# The smartpqi driver, as a pHCI driver, must specify the vHCI class it
+# belongs to(scsi_vhci).
+#
+ddi-vhci-class="scsi_vhci";
+
+#
+# By default MPxIO support is enabled. Setting this value to 0 will
+# disable that support.
+#
+enable-mpxio=1;
```

Comment:
I would prefer zero-by-default, and behavior more closely matching the other driver that seems to have been used as the "basis" for this one -- mptsas -- which uses "disable-mpxio" and is set to false by default.